src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/locations/LocationsMigrationFilter.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.locations
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Filter specifying what should be migrated from NOMIS to the Non-Associations service")
+data class LocationsMigrationFilter(
+
+  @Schema(
+    description = "Only include non-associations created on or after this date",
+    example = "2020-03-23",
+  )
+  val fromDate: LocalDate? = null,
+
+  @Schema(
+    description = "Only include non-associations created before or on this date",
```

Comment:
non-assoc

Reply:
thanks mike, as it happens ill probably just delete the filter next time as I dont think itll be needed